scripts/release/Makefile.linux

```diff
@@ -0,0 +1,32 @@
+
+NOMAD_VERSION = 0.9.0-dev
```

Comment:
can we make a `VERSION` textfile in the root of the repo that we read in places that need a version? there's a few places where this is needed now and it gets confusing p quickly.

Reply:
This is the version of the upcoming release that will update root/CHANGELOG/version.go and to be updated through an env-var.  I can make it empty and have it fail if it's not set if that makes life a bit easier.